conjure-java-core/src/integrationInput/java/endpointerrors/com/palantir/product/TestErrors.java

```diff
@@ -160,4 +160,57 @@ public NotFoundSerializableError error() {
             return error;
         }
     }
+
+    /**
+     * This utility class is provided to make creating error-specific exceptions easier for tests. It should not be used
+     * outside of tests!
+     */
+    public static final class ComplicatedParametersTestUtility {
```

Comment:
Another approach would be to have one `TestUtility` class per `XErrors` class. Makes the "Test" part of the name a bit more prominent.
I don't have strong feelings either way.

Reply:
Think I'll merge with the current state. With the approach where we have one test utility, I think we'd have to name each method like `createXSerializableError`. I think it'd be slightly easier to search for usages of places folks have created these errors by having all the constructors have the same name.